source/agora/node/Validator.d

```diff
@@ -222,6 +231,9 @@ public class Validator : FullNode, API
 
         if (this.enroll_man.isEnrolled(this.utxo_set.getUTXOFinder()))
             this.nominator.startNominatingTimer();
+
+        if (this.config.admin.enabled)
+            setTimer(0.seconds, &admin_interface.start, false);  // asynchronous
```

Comment:
```suggestion
            setTimer(0.seconds, &this.admin_interface.start, Periodic.No);  // asynchronous
```

Is this used as a timer or as a `schedule`? If it's used as a schedule please add a comment.

Reply:
Use as timer.
admin is the same as the drive lifecycle of the current node.
If admin controls the operation of the node or if it is Initialization, it may be different.

Reply:
> Use as timer.

That doesn't make sense. Why would you call `start()` multiple times?

Reply:
You set the timer for 0 seconds, right? And there is no repeating. That means it's a `schedule` call. So it should be documented like that.

Reply:
I guess I was mistaken. Okay, I'll fix it. Thank you.

Reply:
Yes please just add a comment `// schedule`